vsts/scripts/tagBuildImagesForRelease.sh

```diff
@@ -6,48 +6,34 @@
 
 set -o pipefail
 
+buildNumber=$BUILD_BUILDNUMBER
+
 function tagBuildImage() {
-    local buildImageRepo="$1"
-    buildNumber=$BUILD_BUILDNUMBER
+    local prodRegistryLatestTagName="$1"
+    local prodRegistrySpecificTagName="$2"
+    local prodRegistryRepoName="oryxmcr.azurecr.io/public/oryx/build"
     sourceBranchName=$BUILD_SOURCEBRANCHNAME
-    buildImageName="$buildImageRepo:Oryx-CI.$buildNumber"
+    buildImageName="oryxdevmcr.azurecr.io/public/oryx/build:Oryx-CI.$buildNumber"
     outFileMCR="$BUILD_ARTIFACTSTAGINGDIRECTORY/drop/images/build-images-mcr.txt"
-    outFileDocker="$BUILD_ARTIFACTSTAGINGDIRECTORY/drop/images/build-images-dockerhub.txt"
 
     echo "Pulling the source image $buildImageName ..."
     docker pull "$buildImageName" | sed 's/^/     /'
-        
-    acrProdRepo="oryxmcr.azurecr.io/public/oryx/build"
-    acrLatest="$acrProdRepo:latest"
-    acrSpecific="$acrProdRepo:$buildNumber"    
-        
-    dockerHubRepoName="oryxprod/build"
-    dockerHubLatest="$dockerHubRepoName:latest"
-    dockerHubSpecific="$dockerHubRepoName:$buildNumber"
 
     echo
-    echo "Tagging the source image with tag $acrSpecific..."
-    echo "$acrSpecific">>"$outFileMCR"
-    docker tag "$buildImageName" "$acrSpecific"
-
-    echo "Tagging the source image with tag $dockerHubSpecific..."
-    echo "$dockerHubSpecific">>"$outFileDocker"
-    docker tag "$buildImageName" "$dockerHubSpecific"
+    echo "Tagging the source image with tag $prodRegistrySpecificTagName..."
+    docker tag "$buildImageName" "$prodRegistryRepoName:$prodRegistrySpecificTagName"
+    echo "$prodRegistryRepoName:$prodRegistrySpecificTagName">>"$outFileMCR"
 
     if [ "$sourceBranchName" == "master" ]; then
-        echo "Tagging the source image with tag $acrLatest..."
-        echo "$acrLatest">>"$outFileMCR"
-        docker tag "$buildImageName" "$acrLatest"
-
-        echo "Tagging the source image with tag $dockerHubLatest..."
-        echo "$dockerHubLatest">>"$outFileDocker"
-        docker tag "$buildImageName" "$dockerHubLatest"
+        echo "Tagging the source image with tag $prodRegistryLatestTagName..."
+        docker tag "$buildImageName" "$prodRegistryRepoName:$prodRegistryLatestTagName"
+        echo "$prodRegistryRepoName:$prodRegistryLatestTagName">>"$outFileMCR"
     else
         echo "Not creating 'latest' tag as source branch is not 'master'. Current branch is $sourceBranchName"
     fi
 
     echo -------------------------------------------------------------------------------
 }
 
-tagBuildImage "oryxdevmcr.azurecr.io/public/oryx/build"
-tagBuildImage "oryxdevmcr.azurecr.io/public/oryx/build-slim"
\ No newline at end of file
+tagBuildImage "latest" "$buildNumber"
+tagBuildImage "slim" "slim-$buildNumber"
```

Comment:
In view of MCR team's comment here I modified it to be like this. What do you guys think of it?
https://github.com/microsoft/mcr/pull/712#pullrequestreview-279210913

so it is going to be `mcr.microsoft.com/oryx/build:slim` (indicating a moving tag or `latest`) and `mcr.microsoft.com/oryx/build:slim-20190823.1`

This feels a bit odd...like the pattern usually with debian image is `debian:stretch` and `debian:stretch-slim`, so it should be like `oryx:build` and `oryx:build-slim`, but I do not know if this is good